python/runtime/model/db.py

```diff
@@ -44,27 +44,53 @@ def _drop_table_if_exists(conn, table):
     conn.execute(sql)
 
 
+# NOTE: MySQL TEXT type can contain 65536 characters at most.
+# We need to limit the max string length of each row.
+MAX_LENGTH_TO_WRITE_PER_ROW = 4096
```

Comment:
How about increase this to 32k to avoid so that large models will not have too many rows.

Reply:
Agree. I would do not in the later PR.